exporter/stackdriver/stats_test.go

```diff
@@ -627,6 +645,77 @@ func TestExporter_createMeasure(t *testing.T) {
 	}
 }
 
+func TestExporter_createMeasure_CountAggregation(t *testing.T) {
+	oldGetMetricDescriptor := getMetricDescriptor
+	oldCreateMetricDescriptor := createMetricDescriptor
+
+	defer func() {
+		getMetricDescriptor = oldGetMetricDescriptor
+		createMetricDescriptor = oldCreateMetricDescriptor
+	}()
+
+	key, _ := tag.NewKey("test-key-one")
+	m, err := stats.Float64("test-measure/TestExporter_createMeasure", "measure desc", "unit")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	v := &view.View{
+		Name:        "test_view_count",
+		Description: "view_description",
+		TagKeys:     []tag.Key{key},
+		Measure:     m,
+		Aggregation: view.Count(),
+	}
+
+	data := view.CountData(0)
+	vd := newTestViewData(v, time.Now(), time.Now(), &data, &data)
+
+	e := &statsExporter{
+		createdViews: make(map[string]*metricpb.MetricDescriptor),
+	}
+
+	var getCalls, createCalls int
+	getMetricDescriptor = func(ctx context.Context, c *monitoring.MetricClient, mdr *monitoringpb.GetMetricDescriptorRequest) (*metric.MetricDescriptor, error) {
+		getCalls++
+		return nil, status.Error(codes.NotFound, "")
+	}
+	createMetricDescriptor = func(ctx context.Context, c *monitoring.MetricClient, mdr *monitoringpb.CreateMetricDescriptorRequest) (*metric.MetricDescriptor, error) {
+		createCalls++
+		if metricType := mdr.MetricDescriptor.Type; metricType != "custom.googleapis.com/opencensus/test_view_count" {
```

Comment:
The usual form for assertion checking in tests (got/want style) is:
```go
if got, want := mdr.MetricDescriptor.Type, "..."; got != want {
  t.Errorf("MetricDescriptor.Type = %q; want %q", got, want)
}
```
It would be best to use this style for all the checks here.

Reply:
Done.